fs/fs.go

```diff
@@ -349,17 +349,16 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 	go func() {
 		rErr := fmt.Errorf("failed to resolve target")
 		for _, s := range src {
-			sociDesc := ocispec.Descriptor{}
-			if desc, ok := c.imageLayerToSociDesc[s.Target.Digest.String()]; ok {
-				sociDesc = desc
+			if sociDesc, ok := c.imageLayerToSociDesc[s.Target.Digest.String()]; ok {
```

Comment:
nit: can we invert this to reduce some nesting?
```
sociDesc, ok := ...
if !ok {
    rErr = error.Wrapf(...)
    continue
}
l, err := fs.resolver.Resovle(...)
...
```

Reply:
+1